_config.yml

```diff
@@ -4,7 +4,7 @@ email: chanindav@gmail.com
 description: >
   Hanzi Writer is a free and open-source javascript library
   for Chinese character stroke order animations and practice quizzes
-baseurl: "/hanzi-writer" # the subpath of your site, e.g. /blog/
+baseurl: "/hanzi-writer/en/index.html" # the subpath of your site, e.g. /blog/
```

Comment:
This shouldn't be changed - it needs to be `/hanzi-writer` since that matches the URL it will be on github (`chanind.github.io/hanzi-writer`).